platform-sdk/docs/core/dynamic-address-book/roster/glossary-of-terms.md

```diff
@@ -0,0 +1,148 @@
+# Roster Glossary of Terms
+![Event Window Terminology](./EventWindowTerminology.drawio.svg)
+![Roster Terminology Diagram](./RosterTerminologyDiagram.drawio.svg)
+## Round Terminology
+
+### Latest Consensus Round
+
+The most recent round to have reached consensus.  Equal to the pending consensus round - 1.
+
+### Pending Consensus Round
+
+The round that the consensus algorithm is currently attempting to reach consensus on. Equal to the latest 
+consensus round + 1.
+
+### Rounds Non-Ancient
+
+A setting. The number of rounds that have reached consensus that are not ancient.
+
+### Ancient
+
+An event is ancient if its `birthRound` is less than or equal to the`latest consensus round` - `rounds non-ancient`.
```

Comment:
Missing a space between "the`latest consensus round`"

Reply:
I removed the spaces completely and changed to camel case to refer to the added term definitions above.